includes/ft_printf_list.sh

```diff
@@ -175,28 +175,28 @@ then
 	CHK_FT_PRINTF_LIST+=("d" "%+d|-42")
 	CHK_FT_PRINTF_LIST+=("d" "%+d|0")
 	CHK_FT_PRINTF_LIST+=("d" "%+d|4242424242424242424242")
-	CHK_FT_PRINTF_LIST2+=("d" "% +d|42")
-	CHK_FT_PRINTF_LIST2+=("d" "% +d|-42")
-	CHK_FT_PRINTF_LIST2+=("d" "%+ d|42")
-	CHK_FT_PRINTF_LIST2+=("d" "%+ d|-42")
-	CHK_FT_PRINTF_LIST2+=("d" "%  +d|42")
-	CHK_FT_PRINTF_LIST2+=("d" "%  +d|-42")
-	CHK_FT_PRINTF_LIST2+=("d" "%+  d|42")
-	CHK_FT_PRINTF_LIST2+=("d" "%+  d|-42")
-	CHK_FT_PRINTF_LIST2+=("d" "% ++d|42")
-	CHK_FT_PRINTF_LIST2+=("d" "% ++d|-42")
-	CHK_FT_PRINTF_LIST2+=("d" "%++ d|42")
-	CHK_FT_PRINTF_LIST2+=("d" "%++ d|-42")
+	CHK_FT_PRINTF_LIST+=("d" "% +d|42")
+	CHK_FT_PRINTF_LIST+=("d" "% +d|-42")
+	CHK_FT_PRINTF_LIST+=("d" "%+ d|42")
+	CHK_FT_PRINTF_LIST+=("d" "%+ d|-42")
+	CHK_FT_PRINTF_LIST+=("d" "%  +d|42")
+	CHK_FT_PRINTF_LIST+=("d" "%  +d|-42")
+	CHK_FT_PRINTF_LIST+=("d" "%+  d|42")
+	CHK_FT_PRINTF_LIST+=("d" "%+  d|-42")
+	CHK_FT_PRINTF_LIST+=("d" "% ++d|42")
+	CHK_FT_PRINTF_LIST+=("d" "% ++d|-42")
+	CHK_FT_PRINTF_LIST+=("d" "%++ d|42")
+	CHK_FT_PRINTF_LIST+=("d" "%++ d|-42")
 
```

Comment:
A 'space' means "print the sign of the number for negative values only".
A '+' means "print the sign of the number for both positive and negative values".

Isn't it inconsistent?